src/main/services/project-service/index.ts

```diff
@@ -29,269 +29,14 @@ interface IProjectHistoryEntry {
   lastOpenedAt: string
 }
 
-const _ProjectService = {
```

Comment:
Should be kept the object and not the class, if im not mistaken. Check with @JoaoGSP